src/BloomExe/MiscUI/ProblemReporterDialog.cs

```diff
@@ -470,10 +469,34 @@ private string GetFullDescriptionContents(bool appendLog)
 			bldr.AppendLine("=Problem Description=");
 			bldr.AppendLine(_description.Text);
 			bldr.AppendLine();
+			GetAdditionalEnvironmentInfo(bldr);
 			GetStandardErrorReportingProperties(bldr, appendLog);
 			return bldr.ToString();
 		}
 
+		private void GetAdditionalEnvironmentInfo(StringBuilder bldr)
+		{
+			if (Book == null)
+				return;
```

Comment:
I guess the thinking is that if there is no book then we aren't interested in the collection settings? I sorta buy that...